lib/sidekiq/form526_backup_submission_process/submit.rb

```diff
@@ -29,6 +29,7 @@ class Submit
         error_message = msg['error_message']
         timestamp = Time.now.utc
         form526_submission_id = msg['args'].first
+        Form526Submission.find_by(id: form526_submission_id).fail_backup_delivery!
```

Comment:
`Form526Submission.find(form526_submission_id)` might be more appropriate if you expect the record to always exist. Otherwise, I suggest adding a safe navigation operator. Same on L83.